wp-includes/sqlite/class-wp-sqlite-translator.php

```diff
@@ -3094,8 +3109,21 @@ private function execute_alter() {
 					if ( ! $token ) {
 						break;
 					}
-					if ( WP_SQLite_Token::TYPE_STRING !== $token->type
-						|| $from_name !== $this->normalize_column_name( $token->value ) ) {
+
+					// Column name can be represented as:
+					//   1. "column_name" (string)
+					//   2. `column_name` (symbol)
+					//   3. column_name   (keyword — not a reserved one)
+					// While CREATE TABLE normalizes column names to strings, it's better to handle all
+					// variants to handle table definitions with names stored in any valid form.
+					$is_column_name = WP_SQLite_Token::TYPE_STRING === $token->type
+						|| WP_SQLite_Token::TYPE_SYMBOL === $token->type
```

Comment:
@JanJakes, I think this change should probably be in a separate PR.

AFAICT, it is unrelated to the main purpose of this PR, and a separate PR would allow us to review the codebase and address other places where we could treat names more thoroughly. Maybe there could be a new helper method to check correctly wherever it is needed.